atd-vze/src/queries/gqlAbstract.js

```diff
@@ -37,6 +37,43 @@ gqlAbstractTableAggregateName (
 }`;
   }
 
+  /**
+   * Returns true if the input string is a valid alphanumeric object key
+   * @param {string} input - The string to be tested
+   * @returns {boolean}
+   */
+  isNestedKey(input) {
+    return input.match(/^[0-9a-zA-Z\-_]+$/) === null;
+  }
+
+  /**
+   * Returns the key for a nested expression
+   * @param {string} exp - The GraphQL expression
+   * @returns {string}
+   */
+  getExpKey = exp => exp.split(/[{} ]+/, 1)[0].trim();
+
+  /**
+   * Returns the value of a nested expression, usually another expression.
+   * @param {string} exp - The GraphQL expressoin
+   * @returns {string}
+   */
+  getExpValue = exp =>
+    exp.substring(exp.indexOf("{") + 1, exp.lastIndexOf("}")).trim();
+
+  /**
+   * Refactors a nested key into `sort` format
+   * @param {string} exp - The nested key (usually a graphql expression)
+   * @returns {string}
+   */
+  sortifyNestedKey = (exp, val) =>
+    this.isNestedKey(exp)
+      ? `${this.getExpKey(exp)}: { ${this.sortifyNestedKey(
+          this.getExpValue(exp),
+          val
+        )} }`
+      : `${exp}: ${val}`;
```

Comment:
This is a recursive expression, basically transforms this:

`location { location_crashes_aggregate { count }}: asc`

into this (example):

`location: { location_crashes_aggregate: {count: asc}}`